lte/gateway/c/session_manager/AmfServiceClient.h

```diff
@@ -48,12 +49,19 @@ class AsyncAmfServiceClient : public GRPCReceiver, public AmfServiceClient {
   bool handle_response_to_access(
       const magma::SetSMSessionContextAccess& response);
 
+  bool handle_notification_to_access(
+      const magma::SetSmNotificationContext& req);
+
  private:
   static const uint32_t RESPONSE_TIMEOUT = 6;  // seconds
   std::unique_ptr<SmfPduSessionSmContext::Stub> stub_;
   void handle_response_to_access_rpc(
       const SetSMSessionContextAccess& response,
       std::function<void(Status, SmContextVoid)> callback);
+
+  void handle_notification_to_access_rpc(
+      const SetSmNotificationContext& notif,
+      std::function<void(Status, SmContextVoid)> callback);
```

Comment:
both of these look quite unnecessary and can be incorporated into functions `handle_response_to_access` and `handle_notification_to_access`

Reply:
Removed other one.